radio/src/gui/colorlcd/channel_bar.h

```diff
@@ -179,10 +196,29 @@ class OutputChannelBar : public ChannelBar
         value = newValue;
         invalidate();
       }
+      int limit = (g_model.extendedLimits ? LIMIT_EXT_MAX : 1000);
+      LimitData* lim = limitAddress(channel);
+      
+      if (GV_IS_GV_VALUE(lim->min - 1000, -limit, 0)) {
+        int ldMin =
+            GET_GVAR_PREC1(lim->min - 1000, -limit, 0, mixerCurrentFlightMode) +
```

Comment:
Maybe it's just me, but I prefer to have the sign (`+`) at the beginning of the number it is adding (helps me read better).